docs/content/src/learn/tutorials/saas-integrations/amazons3/working-with-s3-bucket/README.md

```diff
@@ -0,0 +1,144 @@
+# Working with Amazon S3 Bucket
+
+## About
+
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the 
+support of connectors. In this guide, we are mainly focusing on connecting to the Amazon Simple Storage Service API to create, store,download, and use data with other services.  
+
+The `wso2/amazons3` module allows you to perform the following operations.
+
+This example explains how to use the S3 client to connect with the Amazon S3 instance and to create a Amazon S3 bucket.
+
+You can find other integrations modules from [wso2-ballerina](https://github.com/wso2-ballerina) Github organization.
+
+## What you'll build
+
+This example explains how to use the Amazon S3 connector to connect with the Amazon S3 API and creating a new bucket on Amazon S3 instance with the provided name.
+
+![exposing Amazon S3 as a service](../../../../../assets/img/import-contacts-into-salesforce-using-ftp.jpg)
+
+## Prerequisites
+
+- [Java](https://www.oracle.com/technetwork/java/index.html)
+- A Text Editor or an IDE
+  > **Tip**: For a better development experience, install one of the following Ballerina IDE plugins: 
+[VSCode](https://marketplace.visualstudio.com/items?itemName=ballerina.ballerina), 
+[IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina)
+- Link to download Ballerina Integrator
```

Comment:
We have a different way of adding the prerequisite segment.
e.g.
<!-- INCLUDE_MD: ../../../../../tutorial-prerequisites.md -->

Reply:
Fixed